config/initializers/devise.rb

```diff
@@ -321,6 +321,7 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'], {}
```

Comment:
@apeksha2697 on you local what are you using GOOGLE_CLIENT_ID , GOOGLE_CLIENT_SECRET values as ?